scipy/signal/tests/test_signaltools.py

```diff
@@ -1313,6 +1346,35 @@ def test_invalid_shapes(self, dt):
         assert_raises(ValueError, correlate, *(a, b), **{'mode': 'valid'})
         assert_raises(ValueError, correlate, *(b, a), **{'mode': 'valid'})
 
+    def test_invalid_params(self):
+        a = [3, 4, 5]
```

Comment:
The class is parametrized over dtypes, so this should read
```
def test_invalid_params(self, dt):
    a = np.asarray([3, 4, 5], dtype=dt)
    ...
```
and ditto for the other tests below.

Reply:
Well the tests don't care about the dtype.  Should I create a separate class, then?